packages/@n8n/nodes-langchain/nodes/vendors/OpenAi/methods/listSearch.ts

```diff
@@ -96,10 +118,10 @@ export async function assistantSearch(
 		const results: INodeListSearchItems[] = [];
 
 		for (const assistant of data || []) {
-			if ((assistant.name as string)?.toLowerCase().includes(filter.toLowerCase())) {
+			if (assistant.name?.toLowerCase().includes(filter.toLowerCase())) {
 				results.push({
-					name: assistant.name as string,
-					value: assistant.id as string,
+					name: assistant.name,
```

Comment:
`name: assistant.name ?? '',` should be here as well

Reply:
Here we know that name is defined because of `assistant.name?.toLowerCase()` bit